azurelinuxagent/common/cgroupconfigurator.py

```diff
@@ -639,6 +664,8 @@ def setup_extension_slice(self, extension_name):
                 if not os.path.exists(extension_slice_path):
                     try:
                         slice_contents = _EXTENSION_SLICE_CONTENTS.format(extension_name = extension_name)
+                        if cpu_quota is not None:
```

Comment:
Good point. Ideal world should not be present. I agree we might gave to handle this. Three options I can think of if it exists
1) leave as it is(no-op) like now - may be not a good idea.
2) Compare the contents and update/remove based on quotas provided. Bit cumbersome and difficult since we have to do it for every slice and all drop-in files .
3) Simple and easy - delete it and invoke same block of code to create.